tests/integration/contract-call/contract-call.spec.ts

```diff
@@ -0,0 +1,180 @@
+import { RouteUrls } from '@shared/route-urls';
+
+import { SendPage } from '../../page-objects/send-form.page';
+import { WalletPage } from '../../page-objects/wallet.page';
+import { BrowserDriver, createTestSelector, selectTestNet, setupBrowser } from '../utils';
+import { SECRET_KEY_2 } from '@tests/mocks';
+import { SettingsSelectors } from '@tests/integration/settings.selectors';
+import { Page } from 'playwright-core';
+
+jest.setTimeout(60_0000);
+jest.retryTimes(process.env.CI ? 2 : 0);
+
+describe('Contract Call test', () => {
+  const BEFORE_EACH_TIMEOUT = 600000;
+
+  let browser: BrowserDriver;
+  let walletPage: WalletPage;
+  let mainPage: WalletPage;
+  let sendForm: SendPage;
+
+  beforeEach(async () => {
+    browser = await setupBrowser();
+    walletPage = await WalletPage.init(browser, RouteUrls.Onboarding);
+    await walletPage.signIn(SECRET_KEY_2);
+    await walletPage.waitForHomePage();
+    // switch to test net
+    await selectTestNet(walletPage);
+
+    // go to the previous page and click sign up again
+    let pages = await WalletPage.getAllPages(browser);
+    mainPage = new WalletPage(pages[1]);
+    await mainPage.clickSignUp();
+
+    await walletPage.page.waitForTimeout(2000);
+    // select the first account from the testnet
+    pages = await WalletPage.getAllPages(browser);
+    let latestPage = pages[pages.length - 1];
+    sendForm = new SendPage(latestPage);
+    await sendForm.waitForPreview('$account1');
+    await sendForm.clickFirstAccount();
+
+    // go back to the main page and click contract call button
+    await mainPage.clickContractCall();
+    await walletPage.page.waitForTimeout(2000);
+    // assign the context of latest page to the sendForm for further testing
+    pages = await WalletPage.getAllPages(browser);
+    latestPage = pages[pages.length - 1];
+    sendForm = new SendPage(latestPage);
+    await sendForm.waitForPreview('$standardFeeSelect');
+  }, BEFORE_EACH_TIMEOUT);
+
+  afterEach(async () => {
+    try {
+      await browser.context.close();
+    } catch (error) {}
+  });
+
+  it('when fees is low', async () => {
```

Comment:
This description isn't clear, what behaviour does "when fee is low" test exactly?

Reply:
I can update this just making sure that contract calls allow Low, Standard, High and Custom without error.